companies/services/sankey.py

```
@@ -0,0 +1,134 @@
+from ..models import SankeyData
+
+logger = logging.getLogger(__name__)
+
+class SankeyDataService:
+    def __init__(self):
+        self.api_key = os.getenv('DART_API_KEY')
+        self.url = "https://opendart.fss.or.kr/api/fnlttSinglAcntAll.json"
+        self.REQUEST_TIMEOUT = 30
+
+    def _safe_float(self, val):
+        if not val or val == '-': return 0.0
```

Ruff (0.14.13)

Multiple statements on one line (colon)

(E701)